examples/io/plot_hysplit.py

```diff
@@ -11,10 +11,10 @@
 import act
 import matplotlib.pyplot as plt
 
-from arm_test_data import DATASETS
+from act.tests import sample_files
 
 # Load the data
-filename = DATASETS.fetch('houstonaug300.0summer2010080100')
+filename = sample_files.EXAMPLE_HYSPLIT
```

Comment:
Here we want to keep the DATASETS for the examples as we are trying to be transparent with what files the examples we used. Sorry for the confusion

Reply:
Just the unittests we want to use the sample files module